project/src/main/java/com/google/autograder/data/Database.java

```diff
@@ -163,33 +164,20 @@ public static String getAssignmentsData(String courseID) {
 
         StringBuilder assignmentsData = new StringBuilder("{\"courseWork\": [");
```

Comment:
This will possibly break main. The assignments are usually nested inside a "courseWork" JSON array so if you don't check all this method's dependents to handle your changes this could break main.

Reply:
I've made that change in the js and when I ran it, the assignments are still showing up. I'll delete this line

Reply:
Okay that's good, if it works fine I'll approve.